src/app/interfaces/interfaces.ts

```diff
@@ -0,0 +1,19 @@
+export interface Guest {
+  name: string;
+  phone: string;
+  gender: string;
+  drunker: boolean;
+  canBeRemoved?: boolean;
+}
+
+export interface ResponseApp {
+  title?: string;
+  guest?: Guest;
```

Comment:
`Guest` what? guest who will be killed by the owner of the apartments or the guest which a landlord is speaking to?